libnavigation-base/src/main/java/com/mapbox/navigation/base/options/NavigationOptions.kt

```diff
@@ -27,7 +27,7 @@ const val DEFAULT_NAVIGATOR_PREDICTION_MILLIS = 1000L
  * @param accessToken [Mapbox Access Token](https://docs.mapbox.com/help/glossary/access-token/)
  * @param locationEngine the mechanism responsible for providing location approximations to navigation
  * @param locationEngineRequest specifies the rate to request locations from the location engine.
- * @param timeFormatType defines time format for calculation remaining trip time
+ * @param timeFormatType deprecated, please use [TripNotificationOptions.timeFormatType]
```

Comment:
I found that this is only needed inside the notification. It could be considered out of scope, but when making the `TripNotificationOptions`, it felt wrong to _not deprecate this_ and move it.

Reply:
This can also be an easy follow up.. so i'm just asking for a second opinion here

Reply:
Removed this change